### PR TITLE
VB-859: Restrict access for Manage Prison Visits to specific prisons

### PR DESCRIPTION
**Pull request by psoleckimoj**

### CODE REVIEW

File: helm_deploy/values-prod.yaml [hunk 1]
```diff
@@ -56,6 +56,7 @@ env:
   COMPLEXITY_OF_NEED_URI: https://complexity-of-need.hmpps.service.justice.gov.uk
   PRISONS_WITH_OFFENDERS_THAT_HAVE_COMPLEX_NEEDS: "AGI,BZI,DHI,DWI,ESI,EWI,FHI,LNI,NHI,PFI,SDI,STI"
   PRISONS_WITH_MANAGE_ADJUDICATIONS_ENABLED: "RNI,WLI"
+  PRISONS_FOR_MANAGE_PRISON_BOOKINGS: "HEI"
```
Comment (petergphillips) [inline helm_deploy/values-prod.yaml:59]: this is slightly confusing - you're enabling it for `HEI` but haven't provided the url so I'm guessing it is disabled

File: backend/controllers/homepage/homepage.ts [hunk 2]
```diff
@@ -190,7 +190,10 @@ const getTasks = ({ activeCaseLoadId, locations, staffId, whereaboutsConfig, key
       heading: 'Manage prison visits',
       description: 'Book, view and cancel a prisoner’s social visits.',
       href: managePrisonVisits.ui_url,
-      enabled: () => managePrisonVisits.ui_url && userHasRoles(['MANAGE_PRISON_VISITS']),
+      enabled: () =>
+        managePrisonVisits.ui_url &&
+        userHasRoles(['MANAGE_PRISON_VISITS']) &&
+        managePrisonVisits.enabled_prisons.split(',').includes(activeCaseLoadId),
```
Comment (petergphillips) [inline backend/controllers/homepage/homepage.ts:196]: would be good to add a test for this change

File: helm_deploy/values-prod.yaml (hunk 1 above)
Comment (psoleckimoj) [reply helm_deploy/values-prod.yaml:59]: `MANAGE_PRISON_VISITS_URL: https://manage-prison-visits.prison.service.justice.gov.uk/` 😕

**Review by petergphillips — APPROVED** [review]
LGTM

File: backend/controllers/homepage/homepage.ts (hunk 2 above)
Comment (psoleckimoj) [reply backend/controllers/homepage/homepage.ts:196]: Looking now

File: helm_deploy/values-prod.yaml (hunk 1 above)
Comment (psoleckimoj) [reply helm_deploy/values-prod.yaml:59]: Already in there so marking as resolved.

**Review by petergphillips — APPROVED** [review]
LGTM

**Pull request merged**